examples/dataclasses/hyperparameters_example.py

```diff
@@ -28,21 +28,15 @@ def load(cls, path: str):
             return cls(**config_dict)
 
 
+parser = ArgumentParser()
 parser.add_arguments(HyperParameters, dest="hparams")
 
 args = parser.parse_args()
 
-hparams: HyperParameters = args.hparams
```

Comment:
Do you avoid doing this type of unpacking because of your type checker? They are fine with MyPy, which I use. I'd rather keep those as they are.

Reply:
TBH I changed this because I thought they added a redundant line to the example, diluting the point that the user should pull the objects from the args object. It's a matter of taste, I guess. I'll revert these changes.